shared/container.go

```diff
@@ -285,6 +285,7 @@ var KnownContainerConfigKeys = map[string]func(value string) error{
 	},
 	"snapshots.schedule.stopped": IsBool,
 	"snapshots.pattern":          IsAny,
+	"snapshots.expire":           IsBool,
```

Comment:
Shouldn't this be `snapshots.expiry` and take some kind of delay (probably supporting `m`, `d`, `w`, `y` type suffixes)?